H/HDF5/build_tarballs.jl

```diff
@@ -6,54 +6,18 @@ const YGGDRASIL_DIR = "../.."
 include(joinpath(YGGDRASIL_DIR, "platforms", "mpi.jl"))
```

Comment:
I thought the idea is that we would remove this line and replace it with a `RuntimeDependency`, but I'm a bit confused as well.

@giordano ?

Reply:
Why replacing that line? That'd break everything. At most you want to try and change https://github.com/JuliaPackaging/Yggdrasil/blob/4d82df7a64f9535fdba24119de35cb4c1eecc999/platforms/mpi.jl#L82 to a runtime dep, but I seem to remember that had some problems.

Reply:
#6417

Reply:
Might work now with https://github.com/JuliaPackaging/BinaryBuilderBase.jl/pull/304 and https://github.com/JuliaPackaging/BinaryBuilder.jl/pull/1273 but haven't tried it

Reply:
I think I'm starting to follow. You're saying we need to edit `Yggdrasil/platforms/mpi.jl` by perhaps reverting [c4069bf](https://github.com/JuliaPackaging/Yggdrasil/commit/c4069bff0e6f082fc5c77369c7ba7c5e6c986f4c).

Reply:
Only if it solves https://github.com/JuliaPackaging/Yggdrasil/pull/6417#issuecomment-1476823499

Reply:
I'm thinking we should leave this for another pull request since fixing that will probably hit multiple recipes.

I'm good to merge this as is now if Erik is good.

Reply:
I don't think https://github.com/JuliaPackaging/Yggdrasil/pull/7908#issuecomment-1879275022 was addressed.

Reply:
@giordano That's the comment regarding OpenMP? What is unresolved there? I think we all agree that OpenMP isn't used and isn't mentioned, and since @mkitti removed the respective dependency all is good?

Apart from this: Yes, we can merge.